File: ts-koa/src/check_service.ts
import * as service from './spec/check_service'

export let checkService = (): service.CheckService => {
    let checkEmpty = async (): Promise<void> => {}

    let checkHeader = async (): Promise<void> => {}

    let checkUrlParams = async (params: service.CheckUrlParamsParams): Promise<void> => {}

    let checkForbidden = async (): Promise<service.CheckForbiddenResponse> => {
        return {status: 'forbidden'}
    }

    let sameOperationName = async (): Promise<service.SameOperationNameResponse> => {
        return {status: 'ok'}
    }

    return {checkEmpty, checkHeader, checkUrlParams, checkForbidden, sameOperationName}
}

File: java-micronaut-moshi/src/main/java/test_service/services/check/CheckService.java
package test_service.services.check;

import java.math.BigDecimal;
import java.time.*;
import java.util.*;
import java.io.*;
import test_service.models.*;

public interface CheckService {
	void checkEmpty();
	void checkEmptyResponse(Message body);
	CheckForbiddenResponse checkForbidden();
	SameOperationNameResponse sameOperationName();
}
